xml/ha_maintenance.xml

```diff
@@ -94,7 +138,7 @@ Node <replaceable>&node2;</replaceable>: standby
    </step>
    <step>
     <para>
-     Stop the &pace; service on that node:
+     Stop the cluster services on that node:
```

Comment:
```suggestion
     Stop the &pace; cluster services on that node:
```

Reply:
I wouldn't say "Pacemaker cluster services" as it's not related entirely on Pacemaker only. There is, for example, Corosync. So Yan Gao recommends to use only "cluster services". I think, that is sufficient. :wink: